src/main/resources/standardTestCases/space.test

```diff
@@ -0,0 +1,11 @@
+name: space
+testFormat: workflowsuccess
+
+files {
+  wdl: space/space.wdl
+}
+
+metadata {
+    workflowName: space
+    status: Succeeded
```

Comment:
this tests the value of the status field in the metadata I believe.
I'm not sure where`testFormat: workflowsuccess` gets its information from. @ruchim  ?